pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -294,7 +294,7 @@ protected String extractClientRequestId(SqlNodeAndOptions sqlNodeAndOptions) {
         ? sqlNodeAndOptions.getOptions().get(Broker.Request.QueryOptionKey.CLIENT_QUERY_ID) : null;
   }
 
-  protected void onQueryStart(long requestId, String clientRequestId, String query, Object... extras) {
+  protected void onQueryRunning(long requestId, String clientRequestId, String query, Object... extras) {
```

Comment:
The fact that we are changing this method's semantics in this PR indicates that the lifecycle method was not well defined  (when it is called and what it can do). Can we add a javadoc defining that?